test/fixtures/discordactions/discordactions.js

```diff
@@ -4,6 +4,9 @@ const groupData = [
   { rolename: "Group 3", roleid: 3 },
 ];
 
+const groupIdle7d = { rolename: "group-idle-7d+", roleid: 4, createdBy: "1dad23q23j131j" };
+const groupIdle = { rolename: "group-idle-7d+", roleid: 4, createdBy: "1dad23q23j131j" };
```

Comment:
Do we need this `groupIdle` too ?
As I can see we are adding a role to people who are Idle for more than 7 days

Reply:
no will remove it